playbooks/roles/mongo_3_0/tasks/main.yml

```diff
@@ -322,7 +322,6 @@
 - name: create mongodb users in a replica set
   mongodb_user:
     database: "{{ item.database }}"
-    login_database: 'admin'
```

Comment:
@denibertovic I used it for the enterprise deploy and is working, except for this argument that is not supported until ansible v2, the replica set configuration is working too, the only problem is that not supports arbiter configuration yet, we have a card for add the support. In enterprise customer deploys we are using only 2 mongo members for the replica set, plus a small machine for the arbiter // cc @mrgnr